release/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,31 @@
+# Release Guidelines for the Onfido JS SDK
```

Comment:
At some point, we decided not to put release guidelines inside of the repo due to the security concerns and put them into the Confluence. What about this one?

Reply:
We could maybe have a separate section in the confluence e.g. RELEASE WITH SCRIPT and keep the manual instructions just in case.

Reply:
These release guidelines don't contain any secret so they can be here